server/routes/application-insights.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express');
+const request = require('xhr-request');
+const router = new express.Router();
+
+const host = 'api.applicationinsights.io';
+
+router.post('/query', (req, res) => {
+  const { apiKey, appId, query, queryTimespan } = req.body;
+
+  if (!apiKey || !appId) {
+    return res.send({ error: 'Invalid request parameters' });
+  }
+
+  let url = `https://${host}/beta/apps/${appId}/query`;
+  if (queryTimespan) url += `?timespan=${queryTimespan}`;
+
+  request(url, {
+    method: 'POST',
+    headers: {
+      'x-api-key': apiKey,
+    },
+    body: {
+      query,
+    },
+    json: true,
+  }, (err, result) => {
+    if (err) {
+      console.log(err);
```

Comment:
Agreed; this was copied from another route so I stuck to the existing pattern. Removed in 15b5862.

Note that there are quite a few more `console.log` statements in the server code -- I'll leave them up to you to triage:

```
$ grep -r 'console.log' server/{routes,index.js,app.js,config,dashboards}
server/routes/auth.js:      console.log(`passport profile: ${profile.email}`);
server/routes/auth.js:  console.log(`Registering auth routes...`);
server/routes/auth.js:  console.log(`Auth routes registered.`);
server/routes/cosmos-db.js:    console.log('Invalid request parameters');
server/routes/cosmos-db.js:      console.log(err);
server/index.js:  console.log(`App listening on port ${PORT}!`);
server/app.js:  console.log(`Request URL: ${req.url}`);
server/dashboards/preconfigured/azure.ts:        console.log(state);
server/dashboards/preconfigured/azure.ts:        console.log(state);
```

Once you've decided what to do with these statements, you can set `"no-console": "true"` in the tslint.json file to enforce that no console log statements are included in future pull requests via the Travis CI build.